LinkAdmin.BFF/appsettings.json

```diff
@@ -0,0 +1,235 @@
+        "Enabled": true,
+        "CallbackPath": "/signin-oauth2"
+      },
+      "OpenIdConnect": {
+        "Enabled": false,
+        "Authority": "",
+        "CallbackPath": "/signin-oidc",
+        "NameClaimType": "email",
+        "RoleClaimType": "roles"
+      }
+    }
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "System": "Warning"
+    }
+  },
+  "TelemetryConfig": {
+    "EnableRuntimeInstrumentation": false,
+    "TraceExporterEndpoint": "",
+    "MetricsEndpoint": "",
+    "TelemetryCollectorEndpoint": "http://localhost:55690"
+  },
+  "Serilog": {
+    "Using": [ "Serilog.Sinks.Console", "Serilog.Sinks.Grafana.Loki" ],
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Warning",
+        "System": "Warning"
+      }
+    },
+    "WriteTo": [
+      { "Name": "Console" },
+      {
+        "Name": "GrafanaLoki",
+        "Args": {
+          "uri": "http://localhost:3100",
+          "labels": [
+            {
+              "key": "app",
+              "value": "Link-BoTW"
+            },
+            {
+              "key": "component",
+              "value": "AdminBFF"
+            }
+          ],
+          "propertiesAsLabels": [ "app", "component" ]
+        }
+      }
+    ]
+  },
+  "AllowedHosts": "*",
+  "ReverseProxy": {
+    "Routes": {
+      "route1": {
+        "ClusterId": "AccountService",
+        "Match": {
+          "Path": "api/account/{**catch-all}"
+        }
+      },
+      "route2": {
+        "ClusterId": "AuditService",
+        "Match": {
+          "Path": "api/audit/{**catch-all}"
+        }
+      },
+      "route3": {
+        "ClusterId": "CensusService",
+        "Match": {
+          "Path": "api/census/{**catch-all}"
+        }
+      },
+      "route4": {
+        "ClusterId": "DataAcquisitionService",
+        "Match": {
+          "Path": "api/data/{**catch-all}"
+        }
+      },
+      "route5": {
+        "ClusterId": "MeasureEvaluationService",
+        "Match": {
+          "Path": "api/measureDef/{**catch-all}"
+        }
+      },
+      "route6": {
+        "ClusterId": "NormalizationService",
+        "Match": {
+          "Path": "api/normalization/{**catch-all}"
+        }
+      },
+      "route7": {
+        "ClusterId": "NotificationService",
+        "Match": {
+          "Path": "api/notification/{**catch-all}"
+        }
+      },
+      "route8": {
+        "ClusterId": "ReportService",
+        "Match": {
+          "Path": "api/report/{**catch-all}"
+        }
+      },
+      "route9": {
+        "ClusterId": "ReportService",
+        "Match": {
+          "Path": "api/reportconfig/{**catch-all}"
+        }
+      },
+      "route10": {
+        "ClusterId": "SubmissionService",
+        "Match": {
+          "Path": "api/tenantsubmission/{**catch-all}"
+        }
+      },
+      "route11": {
+        "ClusterId": "SubmissionService",
```

Comment:
Is this supposed to be TenantService cause that is responding to /api/facility

Reply:
good catch, fixed!